homeassistant/components/epic_games_store/const.py

```diff
@@ -0,0 +1,24 @@
+"""Constants for the Epic Games Store integration."""
+
+DOMAIN = "epic_games_store"
+
+CONF_LANGUAGE = "language"
```

Comment:
```suggestion
```

Reply:
Done in 724ce255805e57ea05a7040f6e19d2b945ff67c7